### DISCUSSION TITLE
Is it possible to output a file without the time?

### DISCUSSION THREAD

**Discussion by Antside** [mifi/lossless-cut#1720] [Support]
So whenever I output a file, let's say the original name of the file is "video.mp4", I get a result of something looking like this "video 04-45-52-00.00.39.903-00.00.59.250.mp4". Is there a way to get rid of this?

**Comment by mifi** [comment]
Yes you can customise output name in the export confirmation dialog

**Comment by Antside** [comment]
I'm probably just stupid but in the exporting settings, I only see that I
can change what is added to the output name, but there is no option to
simply select "nothing". Could you send a screenshot of where the option is
located or something? Sorry again for hassle

**Comment by Antside** [comment]
Fuck ok I found it, it was the little icon next to the export button, I
thought it was in the settings I'm sorry and thank you for your help

**Comment by Antside** [comment]
Ok wait sorry but I have one last question. Why is it impossible to set the
output name of the file so that it is the same as the original? Is it
possible to simply cut a video and have it overwrite the previous file,
with the same name and everything?

**Reply by mifi** [reply]
this is because it can make it easy to accidentally overwrite people's input file, thus losing it forever. also i don't think it's possible to delete a file that is currently open on Windows (it is locked). for more discussions, see https://github.com/mifi/lossless-cut/issues/96

**Comment by Antside** [comment]
Alright, thanks a lot for answering my questions
